stmt/stmts/insert.go

```diff
@@ -382,15 +399,18 @@ func (s *InsertValues) initDefaultValues(ctx context.Context, t table.Table, row
 		}
 
 		if mysql.HasAutoIncrementFlag(c.Flag) {
-			if recordID, err = t.AllocAutoID(); err != nil {
-				return 0, errors.Trace(err)
+			// The auto_increment column is not primary key, don't set
```

Comment:
Why add this comment? I do not see the relevance between the comment and the changes.